test/python/dagcircuit/test_dagcircuit.py

```diff
@@ -739,6 +739,30 @@ def test_quantum_predecessors(self):
             or (isinstance(predecessor2, DAGInNode) and isinstance(predecessor1.op, Reset))
         )
 
+    def test_classical_predecessors(self):
+        """The method dag.classical_predecessors() returns predecessors connected by classical edges"""
+
+        self.dag.apply_operation_back(Measure(), [self.qubit1, self.clbit1], [])
+
+        predecessor_measure = self.dag.classical_predecessors(self.dag.named_nodes("measure").pop())
+
+        predecessor1 = next(predecessor_measure)
```

Comment:
Looking at other tests, I think the `assertRaises(StopIteration)` is a good idea to control that there is no wrong length in the result
```
        with self.assertRaises(StopIteration):
            next(predecessor_measure)
```